test/db/tools/correct_cmd_i.txt

```diff
@@ -0,0 +1,41 @@
+[Sections]
```

Comment:
Please upload this here: https://github.com/rizinorg/rizin-testbins/tree/master/other/rz-diff

Reply:
Ok https://github.com/rizinorg/rizin-testbins/pull/35